client/src/components/User/APIKey/APIKeyItem.vue

```diff
@@ -59,6 +59,11 @@ const deleteKey = () => {
                                 :text="item.key"
                                 title="Copy key" />
                         </b-input-group-text>
+
+                        <b-button v-b-tooltip.hover title="Show/hide key" @click="hover = !hover">
```

Comment:
Reaaaally nitpicking... but how about renaming `hover` to `reveal` or something similar? :)